src/neo/Cryptography/Base58.cs

```diff
@@ -45,10 +45,13 @@ public static byte[] Base58CheckDecode(this string input)
         }
 
         /// <summary>
-        /// Converts a byte array to its equivalent <see cref="string"/> representation that is encoded with base-58 digits. The encoded <see cref="string"/> contains the checksum of the binary data.
+        /// Converts a byte array to its equivalent <see cref="string"/>
+        /// representation that is encoded with base-58 digits.
+        /// The encoded <see cref="string"/> contains the checksum of the binary data.
         /// </summary>
         /// <param name="data">The byte array to convert.</param>
-        /// <returns>The <see cref="string"/> representation, in base-58, of the contents of <paramref name="data"/>.</returns>
+        /// <returns>The <see cref="string"/> representation,
+        /// in base-58, of the contents of <paramref name="data"/>.</returns>
```

Comment:
If there is a line break in the middle of the comment, the xml tags should be on a separate line.